dbml-homepage/docs/js-module/connector.md

```diff
@@ -0,0 +1,56 @@
+---
+title: '@dbml/connector'
+---
+
+[![NPM](https://img.shields.io/npm/v/@dbml/connector)](https://www.npmjs.com/package/@dbml/connector)
+
+This package is responsible for connecting to a database on your local environment and fetch its schema information.
+
+## Supported Databases
+
+- PostgreSQL
+- MySQL
+- MSSQL
+
+## Prerequisites
+
+Before you begin, ensure you have met the following requirements:
+
+- **Node.js**: You need to have Node.js version 18.0.0 or higher installed.
```

Comment:
There is only one requirement her. I suggest:

```
## Prerequisites

- Require Nodejs version 18.0.0 or higher.
```

Reply:
Fixed in commit https://github.com/holistics/dbml/pull/615/commits/790438dd7ef00ab26cbd219a7ffe50634ddc5923